modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -75,6 +56,14 @@ void normAssert(cv::InputArray ref, cv::InputArray test,
     EXPECT_LE(normInf, lInf) << comment;
 }
 
+namespace IE = InferenceEngine;
+
+void setNetParameters(IE::CNNNetwork& net) {
+    auto &ii = net.getInputsInfo().at("data");
+    ii->setPrecision(IE::Precision::U8);
+    ii->setLayout(IE::Layout::NHWC);
```

Comment:
This is a mis-use. No need to set `NHWC` when you do `SetBlob()` (and you do).

Reply:
Agreed. Sorted it out and removed